plugins/entity-feedback/src/components/FeedbackResponseTable/FeedbackResponseTable.tsx

```diff
@@ -91,7 +99,36 @@ export const FeedbackResponseTable = (props: FeedbackResponseTableProps) => {
         </>
       ),
     },
-    { title: 'Comments', field: 'comments', width: '40%' },
+    {
+      title: 'Comments',
+      field: 'comments',
+      width: '40%',
+      render: (response: ResponseRow) => {
+        // Check if comment is a stringified object
+        let parsedComment;
+        try {
+          parsedComment = response?.comments && JSON.parse(response.comments);
```

Comment:
Maybe we should import the `Comment` interface here do something like 
`parsedComment = response?.comments && JSON.parse(response.comments) as Comment;` 
to be a bit more explicit on the expected type